lib/handlers/listing.js

```diff
@@ -44,19 +44,28 @@ function getListingsByUserID(user_id, callback) {
 }
 exports.getListingsByUserID = getListingsByUserID;
 
-function getListings(listing_type, callback) {
+function getListings(listing_type, pageNum, callback) {
     var params = {
         'integrity.visible': true,
         'integrity.flag': {'$ne':true }
     };
     if (listing_type != null) {
         params['type'] = listing_type;
     }
-    listing.model
-        .find(params)
-        .sort({lastRefreshed: 'desc'})
-        .limit(Consts.MAX_LISTING_RESULTS)
-        .exec(callback);
+    if (listing_type == 'SubletListing'){
```

Comment:
We could do that - but right now given that we don’t have any sublets I’m not worried about it. I don’t anticipate us surpassing 100 sublets in the near future and if we did changing that would be quick. 
> On Sep 10, 2016, at 1:45 PM, Brian Cefali <notifications@github.com> wrote:
> 
> In lib/handlers/listing.js <https://github.com/TuftsTusk/ringmaster/pull/18#discussion_r78281019>:
> 
> >      var params = {
> >          'integrity.visible': true,
> >          'integrity.flag': {'$ne':true }
> >      };
> >      if (listing_type != null) {
> >          params['type'] = listing_type;
> >      }
> > -    listing.model
> > -        .find(params)
> > -        .sort({lastRefreshed: 'desc'})
> > -        .limit(Consts.MAX_LISTING_RESULTS)
> > -        .exec(callback);
> > +    if (listing_type == 'SubletListing'){
> Should we just make the limit massive then?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/TuftsTusk/ringmaster/pull/18/files/a10bf340a47edfc004aa498629e4c1855a685aba#r78281019>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ACtOnTboC3PWqt19ypZOzvvwRmYNaodHks5qouyogaJpZM4J4TmX>.
>